ppr-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "0.3.58",
+  "version": "0.3.59",
```

Comment:
Will need to `npm i` to update your plock as well. 
That being said, what version of Node is your system?
I noticed quite a large diff in the plock last time, which was probably due to the node version.

Reply:
v16.17.0, will update plock

Reply:
I believe we are using Node version `16.14.2` for specific reasons, fairly certain it was certain vulnerabilities in Node between a certain version set.
Could you roll back your Node Version before committing a new plock?
I can't recall the exact details but i imagine Omid might recall, there was a ticket awhile back to ensure this was the version we were on across the projects.

Reply:
Will do, thanks for clarification